client/src/components/StyleBubble.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+
+function StyleBubble(props) {
+  
+  const { style, setSelectedStyle } = props;
+  
+  function handleClick() {
+    console.log(style);
+    setSelectedStyle([style]);
+    
+  }
+
+
+  if (style !== undefined) {
```

Comment:
can be just
`if (style)`
as long as `style` will never be a falsey value other than undefined